src/main/java/io/vertx/ext/consul/v1/Catalog.java

```diff
@@ -0,0 +1,4 @@
+package io.vertx.ext.consul.v1;
```

Comment:
@vietj for the new API we propose package `io.vertx.consul.v1`, without `ext`. Is it OK?

Reply:
without ext is better, why v1 ?

Reply:
This is the official part of the Consul API versioning, even in packages